gradle/run.gradle

```diff
@@ -37,8 +37,31 @@ testClusters {
   }
 }
 
+testClusters {
+  runMultiNodeTask {
+    testDistribution = 'archive'
+  }
+}
+
+final String nodes = findProperty('nodeCount') ?: "3"
+final String zones = findProperty('zoneCount') ?: "1"
+final int nodeCount = Integer.valueOf(nodes)
+final int zoneCount = Integer.valueOf(zones)
+
+testClusters {
+  runMultiNodeTask {
+    testDistribution = 'archive'
+    numberOfZones = zoneCount
+    numberOfNodes = nodeCount
+  }
+}
+
 tasks.register("run", RunTask) {
-  useCluster testClusters.runTask;
+  if (project.hasProperty('nodeCount')) {
+    useCluster testClusters.runMultiNodeTask;
```

Comment:
this can disrupt other plugins which use the same property. Should we use `zoneCount` here and have `runMultiZoneTask` instead ?